components/core/src/fs.rs

```diff
@@ -201,13 +201,39 @@ pub fn find_command(command: &str) -> Option<PathBuf> {
                 if candidate.is_file() {
                     return Some(candidate);
                 }
+                else {
+                    match find_command_with_pathext(&candidate) {
```

Comment:
The match here would be unnecessary if the `;` on line 202 was removed along with the `None` on line 211. `find_command_with_pathext` already returns an `Option` so if you just call that without the `;` appended to the end you'll get the return value you're looking for!